spec/event_tag_utils_spec.rb

```diff
@@ -38,10 +38,16 @@
     end
     it 'should return nil if event tags contains the revenue with a string value' do
       event_tags = {
-        'revenue' => '65536',
+        'revenue' => 'string',
       }
       expect(Optimizely::Helpers::EventTagUtils.get_revenue_value(event_tags)).to be_nil
     end
+    it 'should return nil if event tags contains the revenue with a string value' do
+      event_tags = {
+        'revenue' => '65536',
+      }
+      expect(Optimizely::Helpers::EventTagUtils.get_revenue_value(event_tags)).to eq('65536')
```

Comment:
I think this should be "..... .to be_nil".

Reply:
https://github.com/optimizely/node-sdk/blob/94f940cff1872ec1023fd9eb2ad4a3781d31d61d/lib/utils/event_tag_utils/tests.js#L37

Reply:
@msohailhussain Can you please confirm? My understanding is that 'revenue' strictly requires an integer, not a numeric string. 
https://github.com/optimizely/php-sdk/blob/master/src/Optimizely/Utils/EventTagUtils.php#L46
https://github.com/optimizely/php-sdk/blob/master/tests/UtilsTests/EventTagUtilsTest.php#L40

Reply:
owais, I believe we don't need to modify this method, it should remain as it was since master branch.